staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go

```diff
@@ -36,9 +36,29 @@ type Checker interface {
 
 // NewChecker creates a new policy checker.
 func NewChecker(policy *audit.Policy) Checker {
+	for i, rule := range policy.Rules {
+		policy.Rules[i].OmitStages = unionStages(policy.OmitStages, rule.OmitStages)
+	}
 	return &policyChecker{*policy}
 }
 
+func unionStages(s1 []audit.Stage, s2 []audit.Stage) []audit.Stage {
+	m := make(map[audit.Stage]struct{})
```

Comment:
why not reuse "k8s.io/apimachinery/pkg/util/sets"

Reply:
I tried, but "k8s.io/apimachinery/pkg/util/sets" only supports int64, string, int, string, etc.
It doesn't work well with our audit.Stage.

Do I miss something?
If we can, I will be happy to use it.

Reply:
How about type convert ?

Reply:
```go
package main

import (
	"fmt"
)

type Stage string

func main() {
	a := Stage("aaaaa")
	b := string(a)
	fmt.Printf(b)
	/*
	s := make([]Stage, 3)
	ss := []string(s)
	fmt.Print(s)
	*/
}
```

Doesn't work as expected.

Reply:
can not convert slice, one element by one

Reply:
Yeah, I don't want to convert them one by one. 
So I wrote this function.